tests/commands/test_member.py

```diff
@@ -70,3 +70,13 @@ def test_one_member():
     assert ret[0] == drgn.Object(MOCK_PROGRAM,
                                  MOCK_PROGRAM.type('int'),
                                  value=1)
+
+
+def test_second_member():
+    line = 'addr global_struct | member ts_voidp'
+    objs = []
+
+    ret = invoke(MOCK_PROGRAM, objs, line)
+
+    assert len(ret) == 1
+    assert ret[0] == MOCK_PROGRAM['global_int'].address_of_()
```

Comment:
I don't feel strongly, but it might be worth a quick comment/hint to the reader, that "ts_voidp" points to the "global_int" variable. It took me a second to understand this line, but it makes sense now that I realize the value of the pointer.